pyartcd/pyartcd/util.py

```diff
@@ -645,3 +644,25 @@ async def get_signing_mode(group: str = None, assembly: str = None, group_config
         group_config = await load_group_config(group=group, assembly=assembly)
 
     return 'unsigned' if group_config['software_lifecycle']['phase'] == 'pre-release' else 'signed'
+
+
+def nightlies_with_pullspecs(nightly_tags: Iterable[str]) -> Dict[str, str]:
```

Comment:
This could benefit from some unit tests. It would also help us document what's expected from this function

Reply:
Added